### DISCUSSION TITLE
WYSIWYG strips audio tag attributes

### DISCUSSION THREAD

**Discussion by iksent** [directus/directus#5007] [Archive]
1. Add custom `audio` tag and set "controls" attribute (this tag doesn't display without `controls` attribute)
![Снимок экрана 2021-04-12 в 23 16 07](https://user-images.githubusercontent.com/32956521/114442464-0266b180-9be6-11eb-8759-3840f215df5a.png)

2. Audio aren't being displayed at Wysiwyg editor (problem №1)\
![Снимок экрана 2021-04-12 в 23 16 27](https://user-images.githubusercontent.com/32956521/114442582-21fdda00-9be6-11eb-9c87-d79912c2527c.png)

3. Save item with this content

4. Open item and take a look at the `audio` tag again: attribute was stripped (problem №2)
![Снимок экрана 2021-04-12 в 23 16 53](https://user-images.githubusercontent.com/32956521/114442862-73a66480-9be6-11eb-99b5-a168cbcc29ae.png)

PS: (With `controls="1"` same result)

---

### Project details
Directus Version: 9.0.0-rc.57
Environment: production
OS: Linux 5.4.0-66-generic
Node: 14.16.0

**Comment by rijkvanzanten** [comment]
I'm fairly sure TinyMCE strips this out as a security "feature" by default. 

You should be able to use `{ extended_valid_elements: "audio" }` (https://www.tiny.cloud/docs/configure/content-filtering/#extended_valid_elements) within the TinyMCE Overrides config to allow `audio` 👍🏻
